lib/ansible/modules/windows/win_format.ps1

```diff
@@ -137,9 +141,14 @@ function Format-AnsibleVolume {
 $ansible_volume = Get-AnsibleVolume -DriveLetter $drive_letter -Path $path -Label $label
 $ansible_file_system = $ansible_volume.FileSystem
 $ansible_volume_size = $ansible_volume.Size
+$ansible_volume_alu = (Get-CimInstance -ClassName Win32_Volume | Where-Object { $_.DeviceId -eq $ansible_volume.Path } | Select-Object BlockSize).BlockSize
```

Comment:
More efficient like this `(Get-CimInstance -ClassName Win32_Volume -Filter "DeviceId = '$($ansible_volume.path.replace('\','\\'))'" -Property BlockSize).BlockSize'